tests/cypress/support/commands.js

```diff
@@ -104,98 +106,106 @@ function hasTopMenuBar(numTopLinks, numChildLinks) {
-      // Each tab can be activated
-      .each(($li) => {
-        cy.wrap($li).click()
-          .should('have.class', 'is-active')
-      })
+function hasTabs (numOfTabs) {
+  cy.get('ul[id="production-tabs"] > li')
+    .should('have.length', numOfTabs)
+    // Each tab can be activated
+    .each(($li) => {
+      cy.wrap($li).click()
+        .should('have.class', 'is-active')
+    })
 }
 
-function hasDeferAttrsCorrectlyApplied() {
-    // Most JS modules have the 'defer' attributes applied to them
-    // Smoke test on one module...
-    cy.get('head script[src*="MadCapAll.js"')
-      .should('have.attr', 'defer')
-    // The following two modules must not have 'defer' attrs applied
-    cy.get('head script[src*="require.min.js"]')
-      .should('not.have.attr', 'defer')
-    cy.get('head script[src*="foundation.6.2.3_custom.js"]')
-      .should('not.have.attr', 'defer')
+function hasDeferAttrsCorrectlyApplied () {
+  // Most JS modules have the 'defer' attributes applied to them
+  // Smoke test on one module...
+  cy.get('head script[src*="MadCapAll.js"')
+    .should('have.attr', 'defer')
+
+  // The following two modules must not have 'defer' attrs applied
+  cy.get('head script[src*="require.min.js"]')
+    .should('not.have.attr', 'defer')
+
+  cy.get('head script[src*="foundation.6.2.3_custom.js"]')
+    .should('not.have.attr', 'defer')
 }
 
-function hasQualtrics() {
-    cy.get('body script[src*="qualtrics.js"]')
-      .should('exist')
+function hasQualtrics () {
+  cy.get('body script[src*="qualtrics.js"]')
+    .should('exist')
 }
 
-function hasCoveoSearchBar() {
-    cy.get('div.magic-box-input input')
-      .should('be.visible')
+function hasCoveoSearchBar () {
+  cy.get('div.magic-box-input input')
+    .should('be.visible')
 }
 
-function hasMadCapSearchBar() {
+function hasMadCapSearchBar () {
   cy.get('form.search').last().as('searchForm')
     .find('input[aria-label="Search Field"]')
     .should('be.visible')
     .type('verify')
     .should('have.value', 'verify')
     .should('be.visible')
+
   cy.get('@searchForm')
     .find('div.search-submit')
     .should('be.visible')
     .click()
+
   cy.url()
-    .should('include', 'Search.htm?q=verify')  
-}
-
-function switchLocale(changeLocaleStr, langName) {
-    // 'langName' is the display text for the lang that the user will switch to
-    // From EN to JA, this is '日本語 (日本)‎'
-    // From JA to EN, this is 'English (United States)‎'
-    // 
-    // 'changeLocaleStr' is the expected string displayed to a user in the target locale.
-    // For EN this is "Change language"
-    // For JA this is "言語の変更"
-    cy.get('button.select-language-button')
-      .should(($el) => {
-        expect($el).to.be.visible
-      })
-      .trigger('mouseover')
-      .find('div.button-icon-wrapper')
-      .should(($el) => {
-        expect($el).to.have.attr('aria-label', changeLocaleStr)
-        expect($el).to.be.visible
-      })
-      .click()
```

Comment:
I'm not clear why its actually flaky, I think `.find` changes current dom element from the button to area label. On real site it's impossible to click on this tooltip 
<img width="194" alt="image" src="https://user-images.githubusercontent.com/4158731/201219561-ec484232-0405-464a-a964-afc2581f3be8.png">
because it disappears when you hover a mouse over it.
My guess is that from script `click` on tooltip actually works as click on switch language icon. And in some cases it fails because tooltip is not active and `click` action does nothing.

Will see in next runs, if we still have flaky tests after this change